homematic/DOCS.md

```diff
@@ -132,6 +132,21 @@ configuration. If you installed Hassio on another distribution of Linux, you nee
 follow the installation guide for the UART USB to setup the UART USB interface on
 your computer.
 
+## BidCos Errors / Coming from raspmatic/pivccu ?
+
+It is recommended to flash the HM-MOD-RPI-PCB again as raspmatic uses a modified firmware as far as we know.
```

Comment:
Sorry, but this is not correct. RaspberryMatic as well as piVCCU uses the latest official firmware for the HM-MOD-RPI-PCB which you can find here: https://github.com/eq-3/occu/tree/master/firmware/HM-MOD-UART

And this is version 2.8.6 (`dualcopro_si1002_update_blhm.eq3`) which enables the HM-MOD-RPI-PCB to communicate with HomeMatic (BidCos-RF) as well as homematicIP devices. So, by referring users to force install version 1.4.1 (`coprocessor_update.eq3`) of the HM-MOD-RPI-PCB the rf module will not be able to communicate with homematicIP devices anymore. This should at least mentioned here.

Reply:
oh good to know @jens-maus. This instruction was just based on pure observations when using the module with RaspberryMatic it wont work with the occu plugin anymore unless the coprocessor_update.eq3 is flashed. I did not try dualcopro_si1002_update_blhm.eq3 as i didnt know about that one. Think ill try that out, maybe this plugin somehow cannot work with that firmware.